lib/src/presentation/pages/home_page.dart

```diff
@@ -1,44 +1,70 @@
 import 'package:flutter/material.dart';
+import 'package:movie_card/src/presentation/pages/popular.dart';
+import 'package:movie_card/src/presentation/widgets/drawer/drawer.dart';
+import 'package:movie_card/src/presentation/widgets/drawer/side_menu_elements.dart';
+import '../widgets/menu/header.dart';
+import '../bloc/movies_bloc.dart';
 import '../../core/util/ui_consts.dart';
-import '../../domain/entity/movie.dart';
 import 'movie_menu.dart';
 import '../widgets/menu/exit_alert.dart';
 import '../widgets/menu/menu_custom_navbar.dart';
 
 class HomePage extends StatefulWidget {
   const HomePage({
-    required this.lastMovie,
     super.key,
-  });
-
-  final Movie lastMovie;
 
-  static const String routeName = "/home";
+  });
 
   @override
   State<StatefulWidget> createState() => _HomePage();
 }
 
-class _HomePage extends State<HomePage> {
+class _HomePage extends State<HomePage> with SingleTickerProviderStateMixin {
   static const int initialIndex = 0;
   int selectedIndex = initialIndex;
+
   late List<Widget> _pages;
   late PageController _pageController;
   static const String backgroundAssets = 'assets/menu_background.png';
+  final MoviesBloc bloc = MoviesBloc();
+  static const int animationPageTransition = 500;
+
+  // For the drawer
+  static bool isSideMenuClosed = true;
+  static late AnimationController animationController;
+  static const int animationDrawerDuration = 200;
 
   @override
   void initState() {
+    bloc.initialize();
     super.initState();
     _pages = <Widget>[
-      MovieMenu(lastMovie: widget.lastMovie),
+      MovieMenu(
+        moviesBloc: bloc,
+      ),
+      Popular(
+        moviesBloc: bloc,
+      ),
+      Popular(
+        moviesBloc: bloc,
+      ),
```

Comment:
remove the trailing commas, in some case when you have only one parameter is not need it

Reply:
Okay, fixed